screensaver.digitalclock/addon.xml

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
 <addon id="screensaver.digitalclock"
 	name="Digital Clock Screensaver"
-	version="5.0.3"
+	version="5.0.4"
 	provider-name="Vojislav Vlasic">
 	<requires>
 		<import addon="xbmc.python" version="2.20.0"/>
```

Comment:
Can you bump this up to 2.25.0 Krypton release.

Reply:
Sure!
Should I send a pull request just for that part, or the whole thing together?

Reply:
sam pr, rebase it as one commit.